src/UKHO.S100PermitService.StubService/StubData/Holdings/response-200-licenceid-50.json

```diff
@@ -1,331 +1,331 @@
-        "cellTitle": "101UN27RED2R27L5C",
-        "latestEditionNumber": "1",
-        "latestUpdateNumber": "2"
+        "datasetName": "101UN27RED2R27L5C.000",
+        "datasetTitle": "101UN27RED2R27L5C",
+        "latestEditionNumber": 1,
+        "latestUpdateNumber": 2
       },
       {
-        "cellCode": "102GE959048208012.h5",
-        "cellTitle": "102GE959048208012",
-        "latestEditionNumber": "21",
-        "latestUpdateNumber": "9"
+        "datasetName": "102GE959048208012.h5",
+        "datasetTitle": "102GE959048208012",
+        "latestEditionNumber": 21,
+        "latestUpdateNumber": 9
       },
       {
-        "cellCode": "102GR30907391012.h5",
-        "cellTitle": "102GR30907391012",
-        "latestEditionNumber": "2",
-        "latestUpdateNumber": "7"
+        "datasetName": "102GR30907391012.h5",
+        "datasetTitle": "102GR30907391012",
+        "latestEditionNumber": 2,
+        "latestUpdateNumber": 7
       },
       {
-        "cellCode": "104EG01_qE4cDLyIu89987Tzbw_20190703T00Z.h5",
-        "cellTitle": "104EG01_qE4cDLyIu89987Tzbw_20190703T00Z",
-        "latestEditionNumber": "11",
-        "latestUpdateNumber": "2"
+        "datasetName": "104EG01_qE4cDLyIu89987Tzbw_20190703T00Z.h5",
+        "datasetTitle": "104EG01_qE4cDLyIu89987Tzbw_20190703T00Z",
+        "latestEditionNumber": 11,
+        "latestUpdateNumber": 2
       },
       {
-        "cellCode": "104EG00_qE4cDLyceTRaG5Tzbw_20190703T00Z.h5",
-        "cellTitle": "104EG00_qE4cDLyceTRaG5Tzbw_20190703T00Z",
-        "latestEditionNumber": "9",
-        "latestUpdateNumber": "8"
+        "datasetName": "104EG00_qE4cDLyceTRaG5Tzbw_20190703T00Z.h5",
+        "datasetTitle": "104EG00_qE4cDLyceTRaG5Tzbw_20190703T00Z",
+        "latestEditionNumber": 9,
+        "latestUpdateNumber": 8
       },
       {
-        "cellCode": "111CR436Nzqb_R49NS-MfUiHLaaVMFLAc_20190703T00Z.h5",
-        "cellTitle": "111CR436Nzqb_R49NS-MfUiHLaaVMFLAc_20190703T00Z",
-        "latestEditionNumber": "23",
-        "latestUpdateNumber": "12"
+        "datasetName": "111CR436Nzqb_R49NS-MfUiHLaaVMFLAc_20190703T00Z.h5",
+        "datasetTitle": "111CR436Nzqb_R49NS-MfUiHLaaVMFLAc_20190703T00Z",
+        "latestEditionNumber": 23,
+        "latestUpdateNumber": 12
       },
       {
-        "cellCode": "111NZ06pbwAj0_h1AApc3dHcL-eyKP9G4GuDepwiEW19QrKA_20190703T00Z.h5",
-        "cellTitle": "111NZ06pbwAj0_h1AApc3dHcL-eyKP9G4GuDepwiEW19QrKA_20190703T00Z",
-        "latestEditionNumber": "7",
-        "latestUpdateNumber": "9"
+        "datasetName": "111NZ06pbwAj0_h1AApc3dHcL-eyKP9G4GuDepwiEW19QrKA_20190703T00Z.h5",
+        "datasetTitle": "111NZ06pbwAj0_h1AApc3dHcL-eyKP9G4GuDepwiEW19QrKA_20190703T00Z",
+        "latestEditionNumber": 7,
+        "latestUpdateNumber": 9
       }
     ]
   },
 
   {
-    "productCode": "P1232",
-    "productTitle": "P1232",
-    "expiryDate": "2026-01-31T23:59:59Z",
-    "cells": [
+    "unitName": "P1232",
```

Comment:
Please remove blanks e.g. line no 89 and other places.

Reply:
Done